services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/settings.py

```diff
@@ -0,0 +1,87 @@
+import logging
+from typing import Optional
+
+from models_library.basic_types import BootModeEnum, PortInt
+from pydantic import BaseSettings, Field, PositiveInt, validator
+
+
+class ServiceSidecarSettings(BaseSettings):
```

Comment:
```suggestion
class DynamicSidecarSettings(BaseSettings):
```

Reply:
MINOR: Why not just ``ApplicationSettings``?  ... 
a) these settings are only used in this folder -> this folder is ONLY for this service, there is not need to distinguish. 
b) there is a single settings per app 
c) independent from the final name of the service
d) this is how is called in the other services